corehq/apps/linked_domain/templates/linked_domain/domain_links.html

```diff
@@ -84,7 +84,7 @@ <h4 class="modal-title">{% trans 'Add Downstream Project Space' %}</h4>
         </div>
         <div class="modal-footer">
           <button type="button" class="btn btn-default" data-dismiss="modal">{% trans 'Cancel' %}</button>
-          <button type="button" class="btn btn-primary" data-dismiss="modal" data-bind="click: addDownstreamDomain">{% trans 'Add' %}</button>
+          <button type="button" class="btn btn-primary" data-dismiss="modal" data-bind="click: addDownstreamDomain, enable: didSelectDomain()">{% trans 'Add' %}</button>
```

Comment:
one thought though, for computed variables is the call `didSelectDomain()` necessary? usually with computeds, you can just use `didSelectDomain` without the `()`...

Reply:
Yeah definitely unnecessary. I was having trouble getting it to work, added that at some point as a sanity check, and never removed it once it was working. Thanks for catching.